tests/restApi/test_get_request.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import pytest
+
+
+BASE_URL = "https://drb-api-qa.nypl.org"
+url = BASE_URL + "/collection/3650664c-c8be-4d07-8d64-2d7003b02048"
+
+def test_get_request():
+    response = requests.get(url)
+    assert response.status_code == 200, f"Expected status code 200, but got {response.status_code}"
+
+def test_collection_name():
+
+    response = requests.get(url)
+    assert response.status_code == 200, f"Expected status code 200, but got {response.status_code}"
+
+    # Parse the JSON response
+    data = response.json()
+
+    # Validate the title in the metadata
+    expected_title = "Works by Ayan"
+    actual_title = data.get("metadata", {}).get("title", "")
+    
+    # Assert that the title is "Works by Ayan"
+    assert actual_title == expected_title, f"Expected title '{expected_title}', but got '{actual_title}'"
+
+
+def test_validate_json_fields():
+   
+    response = requests.get(url)
+    assert response.status_code == 200, f"Expected status code 200, but got {response.status_code}"
+
+    # Parse the JSON response
+    data = response.json()
+
+    # Validate "numberOfItems" is 2
+    assert data.get('metadata', {}).get('numberOfItems') == 2, f"Expected 'numberOfItems' to be 2, but got {data.get('metadata', {}).get('numberOfItems')}"
```

Comment:
What happens if the `numberOfItems` changes?

Reply:
I haven't started UPDATE calls yet. Will be addressed in future PRs
For now it won't change as I have created this collection only for testing